packages/validator/src/slashingProtection/attestation/index.ts

```diff
@@ -147,7 +147,7 @@ export class SlashingProtectionAttestationService {
     // Pre-compute and store lower-bound
     const minSourceEpoch = minEpoch(attestations.map((attestation) => attestation.sourceEpoch));
     const minTargetEpoch = minEpoch(attestations.map((attestation) => attestation.targetEpoch));
-    if (minSourceEpoch != null && minTargetEpoch != null) {
+    if (minSourceEpoch !== null && minTargetEpoch !== null) {
```

Comment:
type of minSourceEpoch is number | null. It can't have the value undefined, making this diff safe